editor/src/messages/tool/tool_messages/select_tool.rs

```diff
@@ -430,6 +430,17 @@ impl Fsm for SelectToolFsmState {
 
 						RotatingBounds
 					} else if intersection.last().map(|last| selected.contains(last)).unwrap_or(false) {
+						if input.keyboard.get(add_to_selection as usize) {
+							let last_selected = intersection.last().unwrap();
```

Comment:
@TrueDoctor is this `unwrap` safe? Also if you can review the rest of this small PR, that'd be appreciated. Thanks.